DigitalLearningSolutions.Web/Views/Supervisor/MyStaffList.cshtml

```diff
@@ -72,34 +72,37 @@
       }
     </div>
   </div>
-  <div class="nhsuk-grid-row">
-    <div class="nhsuk-grid-column-full">
-      <form method="post">
-        <label class="nhsuk-label nhsuk-u-margin-top-6" for="add-user-hint">
-          <h2>Add a member of staff</h2>
-        </label>
-        <div class="nhsuk-form-group">
-          <div class=" sort-select-container">
-            <vc:text-input asp-for="DelegateEmail"
-                           label="User email address"
-                           populate-with-current-value="false"
-                           type="text"
-                           spell-check="false"
-                           hint-text="Provide the work email address of a member of staff to add to your supervision list."
-                           autocomplete="email"
-                           css-class="nhsuk-input" />
+  @if (!Model.IsNominatedSupervisor)
+  {
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <form method="post">
+          <label class="nhsuk-label nhsuk-u-margin-top-6" for="add-user-hint">
+            <h2>Add a member of staff</h2>
+          </label>
+          <div class="nhsuk-form-group">
+            <div class=" sort-select-container">
+              <vc:text-input asp-for="DelegateEmail"
+                             label="User email address"
+                             populate-with-current-value="false"
+                             type="text"
+                             spell-check="false"
+                             hint-text="Provide the work email address of a member of staff to add to your supervision list."
+                             autocomplete="email"
+                             css-class="nhsuk-input" />
+            </div>
           </div>
-        </div>
-        <button class="nhsuk-button" type="submit" asp-action="AddSuperviseDelegate">
-          Submit
-        </button>
-        @Html.HiddenFor(m => m.SearchString)
-        @Html.HiddenFor(m => m.SortBy)
-        @Html.HiddenFor(m => m.SortDirection)
-        @Html.HiddenFor(m => m.Page)
-      </form>
+          <button class="nhsuk-button" type="submit" asp-action="AddSuperviseDelegate">
+            Submit
+          </button>
+          @Html.HiddenFor(m => m.SearchString)
+          @Html.HiddenFor(m => m.SortBy)
+          @Html.HiddenFor(m => m.SortDirection)
+          @Html.HiddenFor(m => m.Page)
+        </form>
+      </div>
     </div>
-  </div>
+  }
   <div class="nhsuk-grid-row">

```

Comment:
`Add multiple staff members` interface elements should also be hidden for `IsNominatedSupervisor`